app/services/universal/helpers.js

```diff
@@ -17,6 +17,18 @@ function incrementIndex(index, inc = 1) {
   return index + inc;
 }
 
+/**
+ * Gets the property value at path of object. If the resolved value is undefined the defaultValue is used,
+ * If no defaultValue is specified return undefined
+ * @param {object} obj The object to query.
```

Comment:
Whoops, we didn't mean to say that all `@param` types should be lowercase, the difference is that `number|string` are primitive types in js, but `Object` is a class so it should be capitalized. We use this handy tool to know how to build good jsdocs https://devhints.io/jsdoc